### PR TITLE
Add Wegue Workshop to docs

### PR DESCRIPTION
**Pull request by JakobMiksch**
- This PR contains a workshop to get started with Wegue
- It builds upon Wegue `v1.0.0`
- The workshop is written as part of the documentation, but could of course be move somewhere else in the future

### CODE REVIEW

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:3]:
```suggestion
This workshop uses Wegue version [`v1.0.0`](https://github.com/meggsimum/wegue/releases/tag/v1.0.0) but works for higher versions with no or little adaptation.
```

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:7]: Maybe adding `Basic knowlegde of Vue.js helps but is not mandatory` would be good.

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:133]:
```suggestion
Open your browser that runs Wegue and refresh the page (typically pressing F5 on your keyboard). Now you should see this web map:
```

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:143]: "The `mapCenter` is a coordinate in used map projection." => "The `mapCenter` is a coordinate in the projection of the map."

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:150]: In the text above `12` is mentioned, but here `11` is used.

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:336]:
```suggestion
The layer list on the top right enables us to switch the airport layer on and off.
```

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:332]:
```suggestion
- `"target": "toolbar"` adds the button for the layer list to the toolbar, alternatively it can be added to a menu using `"target": "menu"`
```

File: docs/workshop.md
Comment (chrismayer) [inline docs/workshop.md:493]:
```suggestion
The changes so far properly registered our module in the Wegue application. To actually activate it in our app we have to adjust our config file. Add this module to the `modules` property of your config file `app/static/app-conf.json`:
```

**Review by chrismayer — COMMENTED** [review]
This is really cool. Thanks @JakobMiksch! 

Some wording stuff, you should have a look at. After that this can be merged.

File: docs/workshop.md
Comment (JakobMiksch) [reply docs/workshop.md:150]: fixed

File: docs/workshop.md
Comment (JakobMiksch) [reply docs/workshop.md]: fixed

File: docs/workshop.md
Comment (JakobMiksch) [reply docs/workshop.md]: done

**Comment by JakobMiksch** [comment]
@chrismayer your approval is needed for merging. Could you approve it if it is okay for you?

**Review by chrismayer — APPROVED** [review]
Thanks fit your ongoing Input.

**Pull request merged**
